### PR TITLE
docs: describe v14 update steps

### PR DESCRIPTION
**Pull request by mhaertwig**
- Dynamic generation of update page
- Describe update to v14

### CODE REVIEW

File: src/showcase/app/app.module.ts [hunk 1]
```diff
@@ -1,15 +1,17 @@
 import { ScrollingModule } from '@angular/cdk/scrolling';
 import { HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
-import { ReactiveFormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
```
Comment (jeripeierSBB) [inline src/showcase/app/app.module.ts:4]: FormsModule is not needed I think?

File: src/showcase/app/howtoupdate/update-steps.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+export interface UpdateStep {
+  from: number;
+  to: number;
+  actions: string[];
+}
+
+const getGeneralActions = (version: number): string[] => [
+  `Update your Angular dependencies to version ${version}.x.x with "--force" flag (due to unmet peer ependencies).<br/>
+        See <a href='https://update.angular.io'>update.angular.io</a> for a step-by-step guide for updating Angular.<br/>
+        <pre>npx @angular/cli@13 update @angular/core@${version} @angular/cli@${version} --force</pre>`,
```
Comment (jeripeierSBB) [inline src/showcase/app/howtoupdate/update-steps.ts:10]:
```suggestion
        <pre>npx @angular/cli@${version} update @angular/core@${version} @angular/cli@${version} --force</pre>`,
```

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Review by jeripeierSBB — APPROVED** [review]
LGTM

File: src/showcase/app/howtoupdate/update-steps.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  {
+    from: 1300,
+    to: 1400,
+    actions: [
+      ...getGeneralActions(14),
+      `Update SBB Angular. Our automatic migration takes care of the changes.
+        <pre>ng update @sbb-esta/angular@14</pre>`,
+      `If your sass files contain any <code>@import</code> to our mixins or sass variables, you have to manually migrate to <code>@use</code>.
+      Example: <p><code>@import '@sbb-esta/angular/styles.scss';</code> becomes <code>@use '@sbb-esta/angular' as sbb;</code>.</p>
+      <p>Additionally, you have to prefix all imported symbols: e.g. <code>pxToRem(3)</code> becomes <code>sbb.pxToRem(5)</code></p>
```
Comment (mhaertwig) [inline src/showcase/app/howtoupdate/update-steps.ts:46]:
```suggestion
      <p>Additionally, you have to prefix all imported symbols: e.g. <code>pxToRem(5)</code> becomes <code>sbb.pxToRem(5)</code></p>
```

**Pull request merged**